src/styles/main.scss

```diff
@@ -708,3 +681,73 @@ Fixing styling for the code example tables in the "Migrating From..." pages
   top: calc(50% - 11px);
   left: 18px;
 }
+
+// CSS styling for the date modifiex X ago
+.quick-info-list {
+  background: #FFFFFF;
+  border: 1px solid #DDDDDD;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin-bottom: 24px;
+  display: flex;
+}
+
+#time-posted-on {
+  padding: 10px 10px 10px 10px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  color: #343434;
+}
+
+.quick-info-list img {
+  margin: 0px 3px 3px 0px;
+}
+
+// badge tags
+.server-version-badge {
+  background-color: white;
+  border: none;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 18px;
+  padding: 0px;
+  cursor: pointer;
+
+  &:not(:nth-child(1)):not(:nth-child(2)):before {
```

Comment:
This can be a bit of a confusing read - can you add a comment above it to explain why we have it?